lib/awful/widget/common.lua.in

```diff
@@ -74,10 +74,14 @@ function common.list_update(w, buttons, label, data, objects)
             }
         end
 
-        local text, bg, bg_image, icon = label(o)
+        local text, bg, bg_image, icon, icon_only = label(o)
```

Comment:
IMHO a table would be best, much easier to extent later on if needed.

For this PR however, I'll use `text ~= nil and text ~= ""`